tests/parm/fd_ufs.yaml

```diff
@@ -1167,7 +1167,39 @@
      - standard_name: inst_tracer_diag_aod
        canonical_units: 1
        description: AOD
+     # IVAI
+     - standard_name: inst_tracer_diag_claie
+       canonical_units: 1
+       description: CLAIE
+     #
+     - standard_name: inst_tracer_diag_cfch
+       canonical_units: 1
+       description: CFCH
+     #
+     - standard_name: inst_tracer_diag_cfrt
+       canonical_units: 1
+       description: CFRT
+     #
+     - standard_name: inst_tracer_diag_cclu
+       canonical_units: 1
+       description: CCLU
      #
+     - standard_name: inst_tracer_diag_cpopu
+       canonical_units: 1
+       description: CPOPU
+     #
+     - standard_name: inst_tracer_diag_coszens
+       canonical_units: 1
+       description: COSZENS
+     #
+     - standard_name: inst_tracer_diag_jo3o1d
+       canonical_units: 1
+       description: JO3O1D
+     #
+     - standard_name: inst_tracer_diag_jno2
+       canonical_units: 1
```

Comment:
Photolysis rate units should be added here too. Do they need to be added anywhere else too?